net-mgmt/lldpd/src/opnsense/scripts/OPNsense/Lldpd/setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+mkdir -p /var/db/collectd
```

Comment:
collectd? :)

Reply:
The script can be ignored, I updated all the relating files and removed it.